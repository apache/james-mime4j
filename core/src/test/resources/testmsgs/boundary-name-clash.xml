<message>
<header>
<field>
Subject:  test</field>
<field>
MIME-Version: 1.0</field>
<field>
Content-Type: multipart/mixed;
	boundary="--boundary.1652331600846930886"</field>
</header>
<multipart>
<body-part>
<header>
<field>
Content-Type: multipart/alternative;
	boundary="--boundary.1652331600846930886-1"</field>
</header>
<multipart>
<body-part>
<header>
<field>
Content-Type: text/plain; charset=utf-8</field>
</header>
<body>
Please see attachment for report Daily_Stats-2022-05-12-0700</body>
</body-part>
<body-part>
<header>
<field>
Content-Type: text/html; charset=utf-8</field>
</header>
<body>
&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"&gt;
&lt;html lang="en"&gt;
	&lt;head&gt;
		&lt;meta http-equiv="content-type" content="text/html; charset=UTF-8"&gt;
	&lt;/head&gt;
	&lt;body&gt;
		&lt;p&gt;Please see attachment for report Daily_Stats-2022-05-12-0700.&lt;/p&gt;
	&lt;/body&gt;
&lt;/html&gt;</body>
</body-part>
</multipart>
</body-part>
<body-part>
<header>
<field>
Content-Type: application/pdf;
	name="Daily_Stats-2022-05-12-0700.pdf"</field>
<field>
Content-Transfer-Encoding: base64</field>
<field>
Content-Disposition: Attachment;
	filename="Daily_Stats-2022-05-12-0700.pdf"</field>
</header>
<body>
JVBERi0xLj4Kc3RhcnR4cmVmCjUzNjEwCiUlRU9GCg==
</body>
</body-part>
</multipart>
</message>
