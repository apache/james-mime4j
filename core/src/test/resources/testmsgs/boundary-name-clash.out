Subject:  test
MIME-Version: 1.0
Content-Type: multipart/mixed;
	boundary="--boundary.1652331600846930886"

----boundary.1652331600846930886
Content-Type: multipart/alternative;
	boundary="--boundary.1652331600846930886-1"

----boundary.1652331600846930886-1
Content-Type: text/plain; charset=utf-8

Please see attachment for report Daily_Stats-2022-05-12-0700
----boundary.1652331600846930886-1
Content-Type: text/html; charset=utf-8

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8">
	</head>
	<body>
		<p>Please see attachment for report Daily_Stats-2022-05-12-0700.</p>
	</body>
</html>
----boundary.1652331600846930886-1--

----boundary.1652331600846930886
Content-Type: application/pdf;
	name="Daily_Stats-2022-05-12-0700.pdf"
Content-Transfer-Encoding: base64
Content-Disposition: Attachment;
	filename="Daily_Stats-2022-05-12-0700.pdf"

JVBERi0xLj4Kc3RhcnR4cmVmCjUzNjEwCiUlRU9GCg==

----boundary.1652331600846930886--
